server/embed/etcd.go

```diff
@@ -818,10 +817,6 @@ func (e *Etcd) pickGRPCGatewayServeContext(splitHTTP bool) *serveCtx {
 }
 
 func (e *Etcd) serveMetrics() (err error) {
-	if e.cfg.Metrics == "extensive" {
-		grpc_prometheus.EnableHandlingTimeHistogram()
-	}
```

Comment:
This part I struggled the most with. For now I'm passing this config down to `v3rpc.Server` as a string argument which doesn't feel great.